server/datastore/mysql/labels.go

```diff
@@ -595,6 +595,7 @@ func (ds *Datastore) applyHostLabelFilters(ctx context.Context, filter fleet.Tea
 	} else if opt.OSSettingsDiskEncryptionFilter.IsValid() {
 		query, params = ds.filterHostsByOSSettingsDiskEncryptionStatus(query, opt, params, enableDiskEncryption)
 	}
+	// TODO: should search columns include display_name (requires join to host_display_names)?
```

Comment:
Based on this related [PR](https://github.com/fleetdm/fleet/pull/15774), it seems like we probably do want this

Reply:
Yeah I think it'd make sense to have it in the list? It's probably weird for the user if we find a host by display name in one place but not in another.